examples/jsm/interactive/SelectionBox.d.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  Camera,
+  Frustum,
+  Mesh,
+  Object3D,
+  Scene,
+  Vector3
+} from '../../../src/Three';
+
+export class SelectionBox  {
+  constructor(camera: Camera, scene: Scene, deep?: number);
+  camera: Camera;
+  collection: Mesh[];
+  deep: number;
+  endPoint: Vector3;
+  scene: Scene;
+  startPoint: Vector3;
+
+  select(startPoint: Vector3, endPoint: Vector3): Mesh[];
```

Comment:
@Mugen87 startPoint and endPoint should be optional.

Reply:
Oh, you are right! Do you want to make an PR with the change?

Reply:
Here it is #17689. Sorry for the delay